modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json

```diff
@@ -419,6 +419,3114 @@
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2022-01-31T00:00:00Z",
+			"goalValue": "19000",
+			"initialReferenceDate": "2022-01-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-12-31T00:00:00Z",
+			"goalValue": "20000",
+			"initialReferenceDate": "2021-12-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-11-31T00:00:00Z",
+			"goalValue": "15000",
+			"initialReferenceDate": "2021-11-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-10-31T00:00:00Z",
+			"goalValue": "10000",
+			"initialReferenceDate": "2021-10-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-09-31T00:00:00Z",
+			"goalValue": "5000",
+			"initialReferenceDate": "2021-09-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-08-31T00:00:00Z",
+			"goalValue": "23000",
+			"initialReferenceDate": "2021-08-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-07-31T00:00:00Z",
+			"goalValue": "8000",
+			"initialReferenceDate": "2021-07-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-06-31T00:00:00Z",
+			"goalValue": "9000",
+			"initialReferenceDate": "2021-06-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-05-31T00:00:00Z",
+			"goalValue": "6000",
+			"initialReferenceDate": "2021-05-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-04-31T00:00:00Z",
+			"goalValue": "13000",
+			"initialReferenceDate": "2021-04-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-03-31T00:00:00Z",
+			"goalValue": "12000",
+			"initialReferenceDate": "2021-03-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-02-31T00:00:00Z",
+			"goalValue": "11000",
+			"initialReferenceDate": "2021-02-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-01-31T00:00:00Z",
+			"goalValue": "19000",
+			"initialReferenceDate": "2021-01-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
```

Comment:
This ERC does not exist. Please change it to "RAY002".

Reply:
Done